ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -605,6 +661,19 @@ objects:
         trap 'rc=$?; if test "${rc}" -eq 0; then touch /tmp/setup-success; else touch /tmp/exit; fi; exit "${rc}"' EXIT
         trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
 
+        while true; do
+          if [[ -f /tmp/shared/exit ]]; then
+            echo "Another process exited" 2>&1
+            exit 1
+          fi
+          if [[ -f /tmp/leased ]]; then
```

Comment:
Should this be `/tmp/shared/lease`?

Reply:
No, `setup` mounts the shared volume directly to `/tmp`, and we haven't bothered to shift it to match the other containers.  A few lines up should be `/tmp/exit` though.

Reply:
Actually, looking at the mount points, this should be fine, but the `/tmp/shared/exit` above should be changed to `/tmp/exit`.

Reply:
No, amazingly the thing is mounted in two different places in the different containers ...